shared/middleware/animation_library_api.rb

```diff
@@ -42,6 +43,25 @@ class AnimationLibraryApi < Sinatra::Base
     end
   end
 
+  #
+  # GET /api/v1/animation-library/level_animations/<filename>
+  # Retrieve an animation that was uploaded by a levelbuilder (for use in level start_animations)
+  #
+  get %r{/api/v1/animation-library/level_animations/(.+)} do |animation_name|
```

Comment:
we could improve the performance of the levelbuilder page if we were to either replace or supplement this endpoint with a bulk endpoint. right now, for N level animations, we make N HTTP requests, but if we were to pass the list of animations to the server in 1 request (or some other bulk amount -- maybe 10 animations per request works best, so we make N/10 server requests)